molecularnodes/io/parse/mda.py

```diff
@@ -176,9 +176,8 @@ def atomic_number(self) -> np.ndarray:
     def vdw_radii(self) -> np.ndarray:
         # pm to Angstrom
         return np.array(
-            [data.elements.get(element,
-                               {'vdw_radii': 100})
-             .get('vdw_radii') for element in self.elements]) * 0.01 * self.world_scale
+            [data.elements.get(element,{}).get(
+                'vdw_radii',100) for element in self.elements]) * 0.01 * self.world_scale
```

Comment:
This is maybe a matter for a different PR, but I was mildly confused why we had the ` * 0.01 * world_scale` before I remembered that we have the vdw_radii in picometres. Something that could probably be handled better in the future.